.travis.yml

```diff
@@ -17,13 +26,22 @@ addons:
     - build-essential
     - libacl1-dev
     - libsystemd-journal-dev
+    - gcc-aarch64-linux-gnu
+    - libc6-dev-arm64-cross
 
 install:
   -
 
 script:
- - ./autogen.sh
  # Build host and fly to ensure we build stage1 init. We don't build everything
  # to avoid downloading large images.
- - ./configure --with-stage1-flavors=host,fly --enable-tpm=auto
- - make unit-check
+ -      if [ "${TARGET}" == "amd64" ]; then
+                ./autogen.sh;
+                ./configure --with-stage1-flavors=host,fly --enable-tpm=auto;
+                make unit-check;
+        elif [ "${TARGET}" == "arm64" ]; then
+                eval "$(GIMME_ARCH=${TARGET} GIMME_CGO_ENABLED=1 ./gimme.local ${TRAVIS_GO_VERSION})";
+                ./autogen.sh;
+                ./configure --host=aarch64-linux-gnu --with-stage1-flavors=host,fly --enable-tpm=auto;
+                make V=2;
```

Comment:
Looking at the rest of this PR, it looks like it is just cross-compiling. I don't think we will be able to run unit-tests there, right?